src/Compilers/CSharp/Portable/BoundTree/BoundDagTest.cs

```diff
@@ -39,5 +43,29 @@ public override int GetHashCode()
         {
             return Hash.Combine(Kind.GetHashCode(), Input.GetHashCode());
         }
+
+#if DEBUG
+        private int _id;
```

Comment:
Consider initializing to `-1` to differentiate from values after the IDs are set.

Reply:
Done